examples/03_connectivity/plot_signal_extraction.py

```diff
@@ -22,7 +24,10 @@
 """
 
 ##############################################################################
-# Retrieve the atlas and the data
+# From images to an ROI-to-ROI correlation matrix
+# -----------------------------------------------
+#
+# Retrieve the atlas and the data.
```

Comment:
May we need to be explicit ? data -> functional data ?

Also, the ```print(........(4D))``` below is misleading I guess. Atlas is not 4D but in 3D. Isn't it ?
If yes, can you correct it ?